### ISSUE TITLE
Poutils 0.10.0 not on PyPI

### ISSUE THREAD

**Issue by JulienPalard** [AFPy/poutils#14]

**Comment by Seluj78** [comment]
Yeah, if you check the actions, I cannot upload it on pypi, and neither can github

**Comment by JulienPalard** [comment]
You're owner of the package on PyPI, how is that possible you can't upload it?

**Comment by JulienPalard** [comment]
ohhh:
```
 <h1>400 The description failed to render in the default format of reStructuredText. See https://pypi.org/help/#description-content-type for more information.</h1>
```

**Comment by JulienPalard** [comment]
Also I totally forgot I had the rights on this project on pypi (even if I created myself) sry.

I fixed the README, and published v0.10.0 and v0.11.0.

**Comment by Seluj78** [comment]
Amazing ! How did you fix it ?
